src/Microsoft.AspNetCore.Blazor.Mono/tools/binaries/illink/illink.runtimeconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "runtimeOptions": {
-    "tfm": "netcoreapp2.0",
+    "tfm": "netcoreapp3.0",
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "2.0.0"
+      "version": "3.0.0*"
```

Comment:
@pranavkm This is to fix https://github.com/aspnet/AspNetCore/issues/6726

I don't know if there are any drawbacks to specifying the version as `3.0.0*`, but it's not enough to specify `3.0.0`, because that won't work with 3.0.0 preview SDKs.

It's OK if this doesn't roll forwards to the final 3.0.0 SDK, because we'll ship a further update specifically for 3.0.0 final.

Reply:
Please let me know if you have any concerns. But in the interest of meeting deadlines, I'm going to merge this and start getting things to flow through right away.